src/index.js

```diff
@@ -2,22 +2,35 @@
 
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
-
+ 
   //... your code goes here
+  const price = product.querySelector('.price span').innerText;
+  const quantity = product.querySelector('.quantity input').value;
+  const subTotal = product.querySelector('.subtotal span');
+  subTotal.innerText = price * quantity;
```

Comment:
Here, you needed to add another variable that would store the value of the `price * quantity` calculation so that you can return that value since we are going to reuse this `updateSubtotal()` function later in our code. 
You could've done it like this:
 ```
const price = product.querySelector('.price span').innerText;
  const quantity = product.querySelector('.quantity input').value;
  const subTotalText = product.querySelector('.subtotal span');
  let subTotal = price * quantity;
  subTotalText.innerHTML = subTotal;
  return subTotal;
```